<launch>
    <param name="number_of_scans_to_average" type="int" value="5"/>
    <param name="minimum_scan_angle" type="int" value="45"/>
    <param name="maximum_scan_angle" type="int" value="225"/>
    <param name="base_frame" type="string" value="axle"/>
    <param name="laser_topic" type="string" value="scan"/>
    <param name="laser_pointcloud" type="string" value="laser_pointcloud"/>

    <include file="$(find rplidar_ros)/launch/rplidar.launch" />

    <node pkg="three_dimensional_lidar_scanner" type="ServoControl.py" name="servo_controller" />
    <node pkg="three_dimensional_lidar_scanner" type="average_scan_server" name="average_scan_server" output="screen" />
    <node pkg="three_dimensional_lidar_scanner" type="three_dimensional_scanner" name="three_dimensional_scanner" output="screen" />
    <node type="rviz" name="rviz" pkg="rviz" args="-d $(find three_dimensional_lidar_scanner)/rviz/view_pointcloud.rviz" />
</launch>